README.md

```diff
@@ -1,12 +1,26 @@
-## RetroAchievements composer package
+[![N|Solid](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRAhk6glux07Qv-MJHXlBmK8LQ0gKALFPp2Yc_UTMGd9AMR_gRLKg)]()
+# RetroAchievements composer package
 
-Install with:
+RetroAchievements composer Package is a ......................................................
+# Open Source
 
-`composer require joestrong/retroachievements`
+  - RetroAchievements composer package is completely open sourced. Any [contribution](https://github.com/joestrong/retroachievements-composer/#contribute) is highly appreciated.
 
-Use in a project:
 
+
+# Installation
+
+RetroAchievements composer package requires [Composer](https://composer.org/) to run.
+
+#### Install the dependencies.
```

Comment:
Can your remove this sentence (this is the package to install, it's not its own dependency)